testsuite/MDAnalysisTests/formats/test_libmdaxdr.py

```diff
@@ -21,376 +21,494 @@
 #
 from __future__ import print_function, absolute_import
 
-from nose.tools import raises
-from numpy.testing import assert_equal, assert_array_equal
-from numpy.testing import assert_array_almost_equal
+import numpy as np
+
 from numpy.testing import assert_almost_equal
+from numpy.testing import assert_array_almost_equal
+from numpy.testing import assert_array_equal
 
 from MDAnalysis.lib.formats.libmdaxdr import TRRFile
-from MDAnalysisTests.datafiles import TRR_single_frame
 from MDAnalysisTests.datafiles import TRR_multi_frame
+from MDAnalysisTests.datafiles import TRR_single_frame
 
 from MDAnalysis.lib.formats.libmdaxdr import XTCFile
-from MDAnalysisTests.datafiles import XTC_single_frame
 from MDAnalysisTests.datafiles import XTC_multi_frame
+from MDAnalysisTests.datafiles import XTC_single_frame
 
-from MDAnalysisTests.tempdir import run_in_tempdir
-import numpy as np
+import pytest
+
+XTC_OFFSETS = np.array([0, 104, 208, 312, 416, 520, 624, 728, 832, 936])
+TRR_OFFSETS = np.array([0, 480, 960, 1440, 1920, 2400, 2880, 3360, 3840, 4320])
+
+
+def test_xtc():
+    f = XTCFile(XTC_single_frame)
```

Comment:
I fixed that. I didn't see any before because the function should clean up nicely and call dealloc. But lets be on the safe side